### PR TITLE
Updated xz to 5.6.3

### PR DESCRIPTION
**Pull request by radarhere**
xz 5.6.3 has been released - https://github.com/tukaani-project/xz/releases/tag/v5.6.3

### CODE REVIEW

File: winbuild/build_prepare.py [hunk 1]
```diff
@@ -185,7 +185,7 @@ def cmd_msbuild(
             cmd_copy(r"src\liblzma\api\lzma\*.h", r"{inc_dir}\lzma"),
         ],
         "headers": [r"src\liblzma\api\lzma.h"],
-        "libs": [r"liblzma.lib"],
+        "libs": [r"lzma.lib"],
```
Comment (hugovk) [inline winbuild/build_prepare.py:188]: Why these changes?

Comment (radarhere) [reply winbuild/build_prepare.py:188]: https://github.com/tukaani-project/xz/releases/tag/v5.6.3
> * MSVC: Rename import library from liblzma.lib to lzma.lib while keeping liblzma.dll name as is. This helps with "pkgconf --msvc-syntax --libs liblzma" because it mungles "-llzma" in liblzma.pc to "lzma.lib".
